docs/application/native/guides/multimedia/image-barcode.md

```diff
@@ -215,8 +215,6 @@ To detect barcodes:
               pkt = NULL;
           }
 
```

Comment:
line 203: Remove this line. (mv_point_s mv_point = {0, 0};)
line 204: Change this line to 'mv_rectangle_s mv_roi = {{0,0}, bardetdata.width, bardetdata.height};'.
line 217: Remove the line. (from the line 217 to 218)
line 224: Remove the line. (from the line 224 to 226)

Reply:
Why didn't you see my previous comment?
I changed this code because the old code made a build break.
The code operates same function as the previous one.
Please don't leave a comment from line 203 to 226.